lua/telescope/config.lua

```diff
@@ -88,6 +88,8 @@ function config.set_defaults(defaults)
   set("file_sorter", sorters.get_fuzzy_file)
 
   set("file_ignore_patterns", nil)
+
+  set("vim_buffers_everywhere", false)
```

Comment:
Oh,.... can I have vim buffers for everywher eexcept grep?

`file_previewer`,
`qflist_previewer`,
`grep_previewer`,
etc.

Reply:
That would require us to `require'previewers'` inside config but previewers already requires'config' for `use_less` and `set_env`. I have never dealt with a circular require in lua. Any Idea?

Reply:
Rather than using a function pointer we could just do the function name. Like `set("file_previewer", 'cat')` and then inside `buitlin.files` do `previewers[config.file_previewer].new(opts)`